### PR TITLE
Refactor navigation button & close components

### LINKED ISSUE #20: Close modal inside portal
When I tried close modal inside portal don't work

### PR DESCRIPTION
**Pull request by CesarCachay**
This pull request:
- Fix the closing modals
- Refactor the header buttons
- Clean unnecessary lines of code
- Solve the error of the welcome view

Close #20

### CODE REVIEW

File: simcase-game/src/components/Header.js [hunk 1]
```diff
@@ -34,41 +34,49 @@ const menuButton = {
...
-  margin: "50 auto",
   padding: 0,
-  width: "50vw",
-  height: "70vh",
+  margin: 0,
+  width: "100vw",
+  height: "100vh",
   borderRadius: 10,
-  background: "84b1ff"
+  backgroundColor: "rgba(0, 0, 255, 0.7)"
```
Comment (test0n3) [inline simcase-game/src/components/Header.js:67]: This one is opacity.
Blur is this one: https://developer.mozilla.org/en-US/docs/Web/CSS/filter

**Review by test0n3 — APPROVED** [review]

File: simcase-game/src/components/Header.js [hunk 2]
```diff
@@ -91,18 +99,14 @@ function Header() {
     setModal(null);
   }
 
-  // const $portal = React.useMemo(() => document.getElementById("portal"), []);
-
   return (
     <header css={headerStyle}>
       <h2
```
Comment (paulotijero) [inline simcase-game/src/components/Header.js:104]: Where declared `headerStyle` I can't find it :thinking: If exist, You could add the style in label `h2` like this:
```jsx
const headerStyle = {
  h2 : {
      width: "33%",
     fontSize: "1.1em"
  }
}
```
Then, should erase the styles intern in label `h2`

File: simcase-game/src/components/Header.js [hunk 3]
```diff
@@ -91,18 +99,14 @@ function Header() {
...
         css={{
           width: "33%",
           fontSize: "1.1em"
         }}
-      >
-        {/* {viewTitle} */}
-      </h2>
+      />
       <a href="/" css={titleLogo}>
```
Comment (paulotijero) [inline simcase-game/src/components/Header.js]: I can't find style `titleLogo`, If don't exist you should erase this

File: simcase-game/src/components/Header.js (hunk 2 above)
Comment (test0n3) [reply simcase-game/src/components/Header.js:104]: Passed some relevant code for that part through Slack.
The idea was to have three elements in the Header component so it centers the University logo. Something I have not done is passing a title for the Header.

I can pass the required changes again.

Comment (paulotijero) [reply simcase-game/src/components/Header.js:104]: Ok I understand that, but this is the other form to declare styles for don't declare in the tag.
This:
```jsx
const headerStyle = {
  ... header styles ...
  h2 : {
      width: "33%",
     fontSize: "1.1em"
  }
}
```
Is the same that this:
```jsx
<header css={headerStyle}>
  <h2 css={{
    width: "33%",
    fontSize: "1.1em"
  }}></h2>
</header>
```

**Review by paulotijero — APPROVED** [review]

**Pull request merged**